### DISCUSSION TITLE
换城市了

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#297] [Announcements]
https://imcbc.cn/202208/wlb-suzhou/

**Comment by penndu** [comment]
过两天要去趟江苏扬州。离开一线城市最主要的就是收入差异，消费习惯要做改变，感觉其它的没什么！

**Comment by penndu** [comment]
在北京买了房，虽然没有落户，但身为房奴感觉已经离不开北京。

**Comment by LGY2333** [comment]
加油bbb，希望苏州的生活能够慢慢治愈你，让你重新拥有热爱的能力

**Comment by caibingcheng** [comment]
> @LGY2333
> 加油bbb，希望苏州的生活能够慢慢治愈你，让你重新拥有热爱的能力
> 

谢谢GY桑

**Comment by caibingcheng** [comment]
> @penndu
> 过两天要去趟江苏扬州。离开一线城市最主要的就是收入差异，消费习惯要做改变，感觉其它的没什么！

我得学会降低自身欲望

**Comment by ryan4yin** [comment]
见见不同的风景，活得更开心一点~

---

另外发现你也有在流利说 A+ 打卡，是跟我的文章有点关系？发现自己的文章对朋友能产生些帮助，我还挺高兴的。

不过我流利说 A+ 就试用了 10 天，感觉它的 AI 语音有点死板，听多了就烦，就把它 Pass 掉了。
我现在仍然在坚持打卡的是「一点英语 APP」以及「薄荷英语 APP」，另外也在看 Moon Palace，这是一本我以前看过的小说，这次打算读读它的原版。

**Comment by caibingcheng** [comment]
> @ryan4yin
> 见见不同的风景，活得更开心一点~
> 
> ---
> 
> 另外发现你也有在流利说 A+ 打卡，是跟我的文章有点关系？发现自己的文章对朋友能产生些帮助，我还挺高兴的。
> 
> 不过我流利说 A+ 就试用了 10 天，感觉它的 AI 语音有点死板，听多了就烦，就把它 Pass 掉了。
> 我现在仍然在坚持打卡的是「一点英语 APP」以及「薄荷英语 APP」，另外也在看 Moon Palace，这是一本我以前看过的小说，这次打算读读它的原版。

哈哈，是因为参考了你的文章然后才决定在流利说上打卡的，不过之前也是计划这份工作期间要锻炼好英语，但是不知道怎么开始，是因为你的文章给了我方向。

我目前是流利说+百词斩，流利说目前对我来说还够使用，所以暂不打算放弃这个。另外就是在discord、telegram、conversationexchange上会找一些外国友人交流，目前还在文字交流的阶段。。。

**Comment by ryan4yin** [comment]
> 哈哈，是因为参考了你的文章然后才决定在流利说上打卡的，不过之前也是计划这份工作期间要锻炼好英语，但是不知道怎么开始，是因为你的文章给了我方向。
> 
> 我目前是流利说+百词斩，流利说目前对我来说还够使用，所以暂不打算放弃这个。另外就是在discord、telegram、conversationexchange上会找一些外国友人交流，目前还在文字交流的阶段。。。

嗯嗯，练习英语交流我觉得第一步得先把口语练好，而口语的第一步是发音。我那篇文章里有给出学习口语的方法~
